ietf/utils/xmldraft.py

```diff
@@ -193,6 +193,9 @@ def render_author_name(author_elt):
         # Use fullname attribute, if present
         fullname = author_elt.attrib.get("fullname", "").strip()
         if fullname:
+            asciifullname = author_elt.attrib.get("asciiFullname", "").strip()
+            if asciifullname and asciifullname != fullname:
```

Comment:
I'm not sure that literal equality is the right test. In xml2rfc, a [method is used to check whether a name is in Latin script](https://github.com/ietf-tools/xml2rfc/blob/bf52f861886a19aacc95938d5ba8ea5b62646785/xml2rfc/uniscripts/__init__.py#L28-L66) and the ascii name is appended if not. That might better reflect when the asciiName is needed.

Reply:
We should probably look at a bunch of drafts and see how often we're going to add extra text to a name already legible to someone who can read the rest of the draft.